README.md

```diff
@@ -1 +1,117 @@
-# sound_manager
+# ad_sound_manager
+
+## Overview
+This node switches the sound to play depending on the state of the system.
+
+This node plays sound in the following order of priority.
+1. Alert people around the ego vehicle once before the ego vehicle departs. The ego vehicle will not depart until the alert is complete.
+2. Keep alerting obstacles around the ego vehicle.
+3. Keep alerting obstacles in the ego vehicle's path.
+4. Notify the cargo receiver once that the ego vehicle has arrived.
+5. Keep informed that the ego vehicle is avoiding obstacles.
+6. Keep informed that the ego vehicle is turning left or right.
+7. Keep requesting an departure permit for on-demand delivery.
+8. Keep alerting people around the ego vehicle while driving.
+9. Notify operator once that the system has booted successfully.
+10. Keep informed that the system is being in shut down sequence.
+
+## Input and Output
+- input
+  - from autoware
+    - `/awapi/vehicle/get/status` : Vehicle status. Refers to the status of the turn signal.
+  - from eve oss
+    - `/autoware_state_machine/state` : State of the system.
+    - `/sound_voice_alarm/audio_res` : Notification that voice alarm playback is complete.
+- output
+  - to eve oss
+    - `/autoware_state_machine/state_sound_done` : Notification that sound playback is complete.
+    - `/sound_voice_alarm/audio_cmd` : Voice alarm playback request.
+    - `/sound_bgm/audio_cmd` : BGM playback requst.
+## Node Graph
+![image](https://user-images.githubusercontent.com/33311630/172429218-87798889-9d9f-454a-b244-fbacbdf9e612.png)
+
+<details>
+
+<summary> plantuml </summary>
+
+```
+
+@startuml
+rectangle "autoware" {
+  usecase "/awapi/awapi_awiv_adapter_node"
+}
+
+rectangle "eve oss" {
```

Comment:
Why does "eve_oss" need to be represented here?

Reply:
Please mark which is this node as discussed separately. Copyright is signed in other file.

Reply:
@MasaakiWatanabeTier4 
Clarified the reference link to the external interface.
https://github.com/eve-autonomy/ad_sound_manager/pull/1/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R21-R34